addons/mail/models/mail_thread.py

```diff
@@ -1609,7 +1609,7 @@ def message_parse(self, message, save_original=False):
     def _message_add_suggested_recipient(self, result, partner=None, email=None, lang=None, reason=''):
         """ Called by _message_get_suggested_recipients, to add a suggested
             recipient in the result dictionary. The form is :
-                partner_id, partner_name<partner_email> or partner_name, reason """
+                partner_id, partner_name<partner_email> or partner_name, reason, default_create_value """
```

Comment:
While we are modifying it, isn't the lang missing in this docstring ?

Reply:
Indeed, I have added it